site/content/contribute/webapp/developer-setup.md

```diff
@@ -26,7 +26,7 @@ Set up your development environment for building, running, and testing the Matte
     ```sh
     export GITHUB_USERNAME=my_username
     mkdir -p $(go env GOPATH)/src/github.com/mattermost
-    git clone https://github.com/$GITHUB_USERNAME/mattermost-webapp.git $(go env GOPATH)/go/src/mattermost/mattermost-webapp
+    git clone https://github.com/$GITHUB_USERNAME/mattermost-webapp.git $(go env GOPATH)/src/mattermost/mattermost-webapp
```

Comment:
Shouldn't this be 
```suggestion
    git clone https://github.com/$GITHUB_USERNAME/mattermost-webapp.git $(go env GOPATH)/src/github.com/mattermost/mattermost-webapp
```
?

Reply:
That's true. Thank you for your suggestions!